src/ui/Progress/GoalProgressIndicator.tsx

```diff
@@ -0,0 +1,102 @@
+import classNames from "classnames";
+import styles from "./styles.module.css";
+import { Icon } from "@sandercamp/ui-components";
+
+type Goal = {
+  id: string;
+  name: string;
+  amount: number;
+  achievedOn: string | null;
+};
+
+type ActiveGoalAmount = {
+  amount: number;
+};
+
+type GoalProgressIndicatorProps = {
+  goals: Goal[];
+  activeKudosMeter: ActiveGoalAmount;
+};
+
+const GoalProgressIndicator = ({
+  goals,
+  activeKudosMeter,
+}: GoalProgressIndicatorProps) => {
+  const goalsDisplay = [...goals].reverse();
```

Comment:
This depends on the assumption that the goals passed into this component are sorted correctly (low to high). I'm not sure if that is the case. I think the API returns an array of Goal objects sorted by id.

Reply:
Now that you say it, I'm not entirely sure iether anymore. Will check!